crypto/objects/obj_dat.c

```diff
@@ -342,14 +347,19 @@ int OBJ_obj2nid(const ASN1_OBJECT *a)
     if (added != NULL) {
         ad.type = ADDED_DATA;
         ad.obj = (ASN1_OBJECT *)a; /* XXX: ugly but harmless */
```

Comment:
Let's git rid of the XXX. Not sure what it means here.

Reply:
Okay, I'll reword the comment.